### PR TITLE
Update WCServletContainerInitializerFilterServletNameMappingTest

### PR DESCRIPTION
**Pull request by pmd1nh**
https://github.com/OpenLiberty/open-liberty/issues/12165

### CODE REVIEW

**Comment by pmd1nh** [comment]
#build       #spawn.fullfat.buckets=com.ibm.ws.webcontainer_fat,com.ibm.ws.webcontainer_fat_lWAS,com.ibm.ws.webcontainer_fat_servlet31,com.ibm.ws.webcontainer_fat_servlet31_webcontainerlWAS,com.ibm.ws.webcontainer.servlet.4.0_fat,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.1,com.ibm.ws.webcontainer.servlet.4.0_fat_toleration.3.0.lWAS

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServletContainerInitializerFilterServletNameMappingTest.java [hunk 1]
```diff
@@ -34,7 +34,7 @@
  * 1. Test the dynamic add servlet after the dynamic filter, which adds a servlet-filter name mapping for that servlet, to make sure no NPE.
  * Also verify that the filter mapping flow is not breaking (i.e continue to the next filter mapping during filter chain build up)
  *
- * 2. Verify the new WARNING message:
+ * 2. If (1) successes, verify the new WARNING message:
```
Comment (pnicolucci) [inline dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServletContainerInitializerFilterServletNameMappingTest.java:37]: Can we change "successes" to "If (1) is successful"?

**Comment by pnicolucci** [comment]
#run-libby-bot

File: dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServletContainerInitializerFilterServletNameMappingTest.java (hunk 1 above)
Comment (pmd1nh) [reply dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/tests/WCServletContainerInitializerFilterServletNameMappingTest.java]: done

**Pull request merged**